src/app/organizer/table-types.ts

```diff
@@ -31,6 +32,8 @@ export interface ColumnDefinition {
   gridWidth?: string;
   /** Header renderer */
   header: React.ReactNode;
+  /** Dropdown label renderer */
```

Comment:
```suggestion
  /**
   * If the column-toggler dropdown should show a different (more verbose?)
   * label, instead of using the header, that content goes here
   */
```
leave a little more info for future code-understanders